src/overlay/test/OverlayTests.cpp

```diff
@@ -2243,6 +2243,23 @@ TEST_CASE("overlay flow control", "[overlay][flowcontrol][acceptance]")
         configs[2].PEER_FLOOD_READING_CAPACITY_BYTES = 0;
         REQUIRE_THROWS_AS(setupSimulation(), std::runtime_error);
     }
+    SECTION("tx batches")
```

Comment:
Just flagging that I'd like to see some more tx batch specific testing, but I assume this will be part of the prod rollout, for testing ony this seems fine.

Reply:
yes, prod rollout will require more rigorous testing. keep in mind that once the rest of the test suite is switched to batching, we'll get a lot of coverage for free.